v2/src/actors/downloadEvents.js

```diff
@@ -13,8 +15,12 @@ class DownloadEventsActor {
     const tenMins = 1500;
     var count = 0;
     const debugMode = true;
-    var testModule = new XMLHttpsRequest();
-    urlS = debugMode ? "https://www.calendarlabs.com/ical-calendar/ics/71/Sweden_Holidays.ics" : "https://www.nordiskamuseet.se/calendar/ical/ical/calendar-nordiska-museet.ics";
+
+    var testModule = new XMLHttpRequest();
```

Comment:
Simple spelling mistake here. The module is called `XMLHttpRequest` not `XMLHttpsRequest`.

https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Using_XMLHttpRequest